docs/rules/symbol-description.md

```diff
@@ -0,0 +1,94 @@
+# Require/Disallow Symbol Description (symbol-description)
+
+The `Symbol` constructor may have optional description:
+
+```js
+var foo = Symbol("foo");
+
+var someString = 'bar';
+var bar = Symbol(someString);
+
+```
+
+
+Using `description` promotes easier debugging.
+
+## Rule Details
+
+This rule allows enforcing or disallowing symbol descriptions.
+
+
+## Options
+
+There are two options for this rule:
```

Comment:
Should I remove options support from the implementation or should I keep it in case some more options are to come?